src/app/components/apps-view/apps-view.component.ts

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-import { Component, OnInit, ViewChild, ElementRef } from '@angular/core';
+import { Component, OnInit, ViewChild, ElementRef} from '@angular/core';
```

Comment:
Hi
I am new to this project and I'd like to offer some suggestions for your consideration.

This import could add whitespace and follow the current style.

```js
import { Component, OnInit, ViewChild, ElementRef } from '@angular/core';
```